docs/content/latest/api/ysql/commands/ddl_create_table.md

```diff
@@ -85,7 +85,13 @@ Using this qualifier will create a temporary table. Temporary tables are only vi
 
 ### Split Into
 
-The `SPLIT INTO` clause specifies the number of tablets that will be created for the table. This is useful for two data center (2DC) deployments. See example below: [Create CDC table specifying number of tablets](#create-cdc-table-specifying-number-of-tablets).
+The `SPLIT INTO` clause specifies the number of tablets that will be created for the table. This is useful for two data center (2DC) deployments. See example below: [Create CDC table specifying number of tablets](#create-cdc-table-specifying-number-of-tablets). 
```

Comment:
I think this should mention more that pre-splitting helps with spread write/read loads on a production cluster.
When you have 30 servers, you use say, 30 tablets, and it will be ready for full write load.

Reply:
Good idea. Done.